tests/MotorControllerTest.cpp

```diff
@@ -219,39 +221,46 @@ namespace etrobocon2025_test {
   TEST(MotorControllerTest, GetRightMotorSpeed)
   {
     MotorController MotorController;
-    int expected = 0;
-    int actual = MotorController.getRightMotorSpeed();
+    double expected = 0.0;
+    double actual = MotorController.getRightMotorSpeed();
     EXPECT_EQ(expected, actual);
-    int speed = 1000;
+    double speed = 1000.0;
     MotorController.setRightMotorSpeed(speed);
-    expected = 1000;
+    double expectedLower = speed - 1;  // getやsetの計算過程でintに丸められた際の許容誤差
     actual = MotorController.getRightMotorSpeed();
-    EXPECT_EQ(expected, actual);
-    speed = -1000;
+    // 　誤差が1以内であれば取得できているとする
+    EXPECT_LE(expectedLower, actual);
+    EXPECT_GE(speed, actual);
+    speed = -1000.0;
     MotorController.setRightMotorSpeed(speed);
-    expected = -1000;
+    double expectedUpper = speed + 1;  // getやsetの計算過程でintに丸められた際の許容誤差
     actual = MotorController.getRightMotorSpeed();
-    EXPECT_EQ(expected, actual);
+    // 　誤差が1以内であれば取得できているとする
+    EXPECT_GE(actual, speed);
+    EXPECT_LE(actual, expectedUpper);
     MotorController.resetRightMotorPower();
   }
 
   // 左車輪のモータの回転速度を取得できるかのテスト
```

Comment:
本当に軽微なヤツ 
さっきの修正の時に気づいてなかった。。。ごめん

右モータの方もおんなじ感じになってるからお願いします

```suggestion
  // 左車輪のモータの線速度を取得できるかのテスト
```